packages/headless/package.json

```diff
@@ -35,7 +35,7 @@
     "@reduxjs/toolkit": "^1.5.0",
```

Comment:
Not sure what happened to the package-lock. If you try installing the latest analytics package, is the diff large?

Reply:
There's definitely something wonky with the package locks, I think they need to be updated more often